draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -111,14 +121,19 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
 document are to be interpreted as described in {{RFC2119}}.
 
 Principle {#principle}
-============
+======================
 
 This section uses the concept of ICE agent as defined in {{RFC8445}}. In the
 remainder of the document, it is assumed that each browsing context (as defined
 in Section 7.1 of {{HTMLSpec}}) has its own ICE agent.
 
 ICE Candidate Gathering {#gathering}
-----------------------------
+------------------------------------
+
+This section outlines how mDNS should be used by ICE agents to conceal local
+IP addresses. Naturally, if the ICE agent does not want to conceal its IPs,
+e.g., if it has a priori knowledge that its addresses are in fact public,
+this processing is unnecessary.
```

Comment:
This statement makes sense, it seems related to some other issue though (
https://github.com/youennf/mdns-ice-candidates/issues/31 and PR https://github.com/youennf/mdns-ice-candidates/pull/62).
Maybe we should remove this change there.

Reply:
The goal here was to indicate who needs to follow which sections. ICE agents that aren't worried about concealing IPs don't need to follow this section.